src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <body>
```

Comment:
There is a duplicated <body> tag on line 16 which should be removed. There should only be one <body> tag in an HTML document.